src/app.yaml

```diff
@@ -22,25 +12,5 @@ handlers:
   http_headers:
     Strict-Transport-Security: "max-age=2592000; includeSubdomains"
 
-- url: /admin/.*
-  script: google.appengine.ext.admin.application
-  login: admin
-
-- url: /superadmin/.*
-  script: main.APP
-  login: admin
```

Comment:
This is one thing I need to fix as right now anyone can access the superadmin routes. Should be able to do it with something in here https://cloud.google.com/appengine/docs/standard/python/migrate-to-python3/migrating-services#user_authentication